resources/lang/en.json

```
@@ -152,6 +152,7 @@
     "giantworldmap": "Giant World Map",
     "europe": "Europe",
     "mena": "MENA",
+    "MiddleEast": "Middle East",
```

I think all map translation keys should be in lowercase.
```suggestion
    "middleeast": "Middle East",
```